examples/quick-start/manifests/demo-application.yaml

```diff
@@ -0,0 +1,694 @@
+# Copyright 2018 Google LLC
```

Comment:
Would personally try to use kustomize to leverage the upstream manifests and reduce/highlight the changes. Should also drop the custom image, which seems to still be there.